transcom/mymove
# [MB-14975] Change calculation for total # days remaining

## [Jira ticket](https://dp3.atlassian.net/browse/MB-14975)

## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

- [ ] Calculates the # “Total days remaining” from Today’s calendar date and the Expiration Date. 
- [ ] If the total days remaining calculates to be <0 days, then reflect “Expired” 
- [ ] If the total days remaining calculates to be =0 days, then reflect “0 days remaining” (eg: If today’s date = Feb 28 and the Expiration Date = Feb 28)
- [ ] Calculated values and Updated labels are reflected on both the SIT Dashboard and the Edit Module

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the Office app
2. Login as a TOO
3. Navigate to `moves/SITEXT/mto`
4. Click edit and change the end date to today's date and save the changes.
    - [x] Remaining days should be 0
5.  Click edit and change total approved days to `59` days and save the changes.
    - [x] Remaining days should be `Expired`

### Frontend


## Screenshots
